src/components/ControllerTop.vue

```diff
@@ -309,7 +309,7 @@ export default {
   data: () => {
     return {
       keymapName: '',
-      width: 0
+      width: 'max-width: 18rem'
```

Comment:
This is the only part of this change I don't understand how it works.

\- @noroadsleft 
*newb who doesn't understand the dark magic that is Vue*

Reply:
yeah, it's basically a component value that is templated into the page. We can move it into the css. I think the original intention was the size of the element was dynamically adjusted based on the width of the text. I got rid of that feature but didn't remove the templating.

Reply:
I'm going to use the CSS you proposed

Reply:
We can revisit this method later on when I understand wtf is going on here.

Reply:
oh yeah for sure. I actually used your css and removed the code.